cmd/queue/main.go

```diff
@@ -197,7 +202,23 @@ func proxyHandler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, tracingEn
 	}
 }
 
-func knativeProbeHandler(healthState *health.State, prober func() bool, isAggressive bool, tracingEnabled bool, next http.Handler) http.HandlerFunc {
+func preferPodForScaledown(logger *zap.SugaredLogger, env config) bool {
+	content, err := ioutil.ReadFile(env.DownwardAPILabelsPath)
```

Comment:
Kinda feels like we can hardcode the path and avoid passing the environment around. If we want it configurable, could we narrow the arguments down to just passing the path?

Reply:
there might be some value in having it configured when creating the pod. will pass in the label only.